modcc/printer/cexpr_emit.cpp

```diff
@@ -262,19 +276,21 @@ void SimdExprEmitter::visit(BinaryExpression* e) {
                 "CExprEmitter: unsupported binary operator " + token_string(e->op()), e->location());
     }
 
-    std::string rhs_name, lhs_name;
+    std::string rhs_name, lhs_name, rhs_pfx, lhs_pfx;
 
     auto rhs = e->rhs();
     auto lhs = e->lhs();
 
     const char *op_spelling = binop_tbl.at(e->op());
     const char *func_spelling = func_tbl.at(e->op());
 
-    if (rhs->is_identifier()) {
-        rhs_name = rhs->is_identifier()->name();
+    if (auto id = rhs->is_identifier()) {
+        rhs_name = id->name();
+        rhs_pfx  = id_prefix(id);
```

Comment:
In the sense of 
```C++
    if (auto id = lhs->is_identifier()) {
        lhs_name = id->name();
        lhs_pfxd  = id_prefix(id) + id->name(); // This is used in instead of lhs_pfx and lhs_name?
    }
```

Reply:
Yes